packages/bpk-component-banner-alert/src/BpkBannerAlertInner.js

```diff
@@ -116,7 +116,7 @@ type Props = {
   onExpandToggle: OnExpandToggleHandler,
 
   // Only relevant when configuration == CONFIGURATION.DISMISSABLE
-  dismissButtonLabel: ?string,
+  dismissButtonLabel: string,
```

Comment:
As `bpk-component-close-button` has the label as required in the component changing this so that flow does not throw and error that null or undefined is not valid